Updates to Overview/Introduction, fix spelling errors (#56)
NCAR/ccpp-doc: CCPPtechnical/source/Overview.rst

The table below lists all parameterizations supported in CCPP public releases and the

> `CCPP Scientific Documentation <https://dtcenter.ucar.edu/GMTB/v5.0.0/sci_doc>`_

Author: @ligiabernardet, I am including a bunch of links to the scientific document below as well. For now I have been pointing to the temporary locations as this makes it easier to ensure that the links are correct, and they can be updated all at once when the scientific documentation is published (it is listed in the issue tracking "to-do" tasks). I think that's a better way to go about it, but if you feel strongly I can make the update to the "future" links now.

> `CCPP Scientific Documentation <https://dtcenter.ucar.edu/GMTB/v5.0.0/sci_doc>`_

Reviewer: OK, this is a reasonable way forward

Finally, *experimental* suites are included in the `ccpp_physics` repository, but are not supported for community use. These may be used experimentally, but they have not been extensively tested, so users should proceed at your their risk.

.. _scheme_suite_table:

.. table:: *Suites supported in the CCPP*

   +---------------------+------------------+------------------+----------------+----------------+----------------+----------------+
   |                     | :bi:`Operational`| :gbi:`Developmental`                                                                 |
   +---------------------+------------------+------------------+----------------+----------------+----------------+----------------+
   | Physics Suite       | GFS_v16          | :g:`GFS_v17_p8`  | :g:`RAP`       |:g:`RRFS_v1beta`| :g:`WoFS`      | :g:`HRRR`      |
   +=====================+==================+==================+================+================+================+================+

>    | **Supported hosts** | **SCM/MRW/SRW**  | :g:`**SCM/MRW**` | :g:`**SCM**`   |:g:`**SCM/SRW**`|:g:`**SCM/SRW**`|:g:`**SCM/SRW**`|

Reviewer: GFSv16 is not supported with MRW
```suggestion
   | **Supported hosts** | **SCM/SRW**  | :g:`**SCM/MRW**` | :g:`**SCM**`   |:g:`**SCM/SRW**`|:g:`**SCM/SRW**`|:g:`**SCM/SRW**`|
```

>    | **Supported hosts** | **SCM/MRW/SRW**  | :g:`**SCM/MRW**` | :g:`**SCM**`   |:g:`**SCM/SRW**`|:g:`**SCM/SRW**`|:g:`**SCM/SRW**`|

Reviewer: There is some issue with the formatting of this line (double asterisks). See https://mike-k-tech-doc.readthedocs.io/en/latest/Overview.html

   +---------------------+------------------+------------------+----------------+----------------+----------------+----------------+
   | Land surface        | Noah             | :g:`Noah-MP`     | :g:`RUC`       | :g:`Noah-MP`   | :g:`Noah-MP`   | :g:`RUC`       |
   +---------------------+------------------+------------------+----------------+----------------+----------------+----------------+
   | Ozone               | NRL 2015         | :g:`NRL 2015`    | :g:`NRL 2015`  | :g:`NRL 2015`  | :g:`NRL 2015`  | :g:`NRL 2015`  |
   +---------------------+------------------+------------------+----------------+----------------+----------------+----------------+
   | Strat H\ :sub:`2`\ O| NRL 2015         | :g:`NRL 2015`    | :g:`NRL 2015`  | :g:`NRL 2015`  | :g:`NRL 2015`  | :g:`NRL 2015`  |
   +---------------------+------------------+------------------+----------------+----------------+----------------+----------------+
   | Ocean               | NSST             | :g:`NSST`        | :g:`NSST`      | :g:`NSST`      | :g:`NSST`      | :g:`NSST`      |
   +---------------------+------------------+------------------+----------------+----------------+----------------+----------------+

> Only the suites that are currently supported in the CCPP are listed in the table. Currently all supported suites use the 2015 Navy Research Laboratory (NRL) `ozone <https://dtcenter.ucar.edu/GMTB/v6.0.0p/sci_doc/html/_g_f_s__o_z_p_h_y_s.html>`_ and `stratospheric water vapor <https://dtcenter.ucar.edu/GMTB/v6.0.0p/sci_doc/html/_g_f_s__h2_o_p_h_y_s.html>`_ schemes,

Reviewer: The future/permanent location for the CCPP v6 SciDoc will be https://dtcenter.ucar.edu/GMTB/v6.0.0, without the "p" in the end. Do you want to fix this now (which will result in broken links) or later?

Author: See my comment above, I'd rather fix it later if that's all right with you.

the `Cooperative Institute for Research in the Environmental Sciences (CIRES) unified gravity wave drag (uGWD) <https://dtcenter.ucar.edu/GMTB/v6.0.0p/sci_doc/html/_g_f_s__u_g_w_p_v0.html>`_ scheme, 
and the `Noah Land Surface Model (LSM) <https://dtcenter.ucar.edu/GMTB/v6.0.0p/sci_doc/html/_g_f_s__n_o_a_h.html>`_.

The five developmental suites are either analogues for current operational physics schemes, or candidates for future operational implementations.

* The GFS_v17_p8 suite is the current (as of June 2022) proposed suite for the next operational GFS implementation (version 17), and features several differences from the GFS_v16 suite, using `Thompson <https://dtcenter.ucar.edu/GMTB/v6.0.0p/sci_doc/html/_t_h_o_m_p_s_o_n.html>`_ microphysics, `saSAS plus Cellular Automata (CA) <https://dtcenter.ucar.edu/GMTB/v6.0.0p/sci_doc/html/_g_f_s__s_a_m_fdeep.html>`_ deep convection, `Unified uGWP <https://dtcenter.ucar.edu/GMTB/v6.0.0p/sci_doc/html/_g_f_s__u_n_i_f_i_e_d__u_g_w_p.html>`_ gravity wave drag, and `Noah Multiparameterization (Noah-MP) <https://dtcenter.ucar.edu/GMTB/v6.0.0p/sci_doc/html/_noah_m_p.html>`_ land surface parameterization.

> * The RAP scheme is similar to the operational Rapid Refresh (RAP) model physics package, and features Thompson microphysics, `Mellor-Yamada-Nakanishi-Niino (MYNN) EDMF <https://dtcenter.ucar.edu/GMTB/v6.0.0p/sci_doc/html/_m_y_n_n_e_d_m_f.html>`_ PBL, `Grell-Freitas (GF) <https://dtcenter.ucar.edu/GMTB/v6.0.0p/sci_doc/html/_c_u__g_f.html>`_ deep convection and shallow convection schemes, RRTMG radiation, `MYNN surface layer (SFL) <https://dtcenter.ucar.edu/GMTB/v6.0.0p/sci_doc/html/_s_f_c__m_y_n_n_s_f_l.html>`_ scheme, `Global Systems Laboratory (GSL) drag <https://dtcenter.ucar.edu/GMTB/v6.0.0p/sci_doc/html/_g_f_s_drag_suite.html>`_ gravity wave drag scheme, and the `Rapid Update Cycle (RUC) Land Surface Model <https://dtcenter.ucar.edu/GMTB/v6.0.0p/sci_doc/html/_r_u_c_l_s_m.html>`_.

Reviewer: I am trying to fix drag appearing twice in the same sentence
```suggestion
* The RAP scheme is similar to the operational Rapid Refresh (RAP) model physics package, and features Thompson microphysics, `Mellor-Yamada-Nakanishi-Niino (MYNN) EDMF <https://dtcenter.ucar.edu/GMTB/v6.0.0p/sci_doc/html/_m_y_n_n_e_d_m_f.html>`_ PBL, `Grell-Freitas (GF) <https://dtcenter.ucar.edu/GMTB/v6.0.0p/sci_doc/html/_c_u__g_f.html>`_ deep convection and shallow convection schemes, RRTMG radiation, `MYNN surface layer (SFL) <https://dtcenter.ucar.edu/GMTB/v6.0.0p/sci_doc/html/_s_f_c__m_y_n_n_s_f_l.html>`_ scheme, `Global Systems Laboratory (GSL) <https://dtcenter.ucar.edu/GMTB/v6.0.0p/sci_doc/html/_g_f_s_drag_suite.html>`_ gravity wave drag scheme, and the `Rapid Update Cycle (RUC) Land Surface Model <https://dtcenter.ucar.edu/GMTB/v6.0.0p/sci_doc/html/_r_u_c_l_s_m.html>`_.
```

was supported).
The CCPP v5.0 release, issued in February 2021, was a major
upgrade to enable use with the UFS Short-Range Weather (SRW) Application and the RRFS_v1alpha suite.
The CCPP v6.0 release, issued in June 2022, was a major upgrade in conjunction with the release of the UFS MRW and SRW v2.0 releases.

.. [#] As of this writing, the CCPP has been validated with two host models: the CCPP
    SCM and the atmospheric component of
    NOAA’s Unified Forecast System (UFS) (hereafter the UFS Atmosphere) that utilizes
    the Finite-Volume Cubed Sphere (FV3) dynamical core.  The CCPP can be utilized both with the
    global and limited-area configurations of the UFS Atmosphere. CCPP v6.0.0 is the latest

>     release compatible with the global UFS Medium-Range Weather Application and the

Reviewer: ```suggestion
    release compatible with the global UFS MRW Application and the
```

>     limited-area UFS Short-Range Weather Application.  The CCPP

Reviewer: ```suggestion
    limited-area UFS SRW Application.  The CCPP
```
